app/components/icons/ArrowDown.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useState } from "react";
+
+function ArrowDown() {
+  const [isActive, setIsActive] = useState(true);
+  return (
+    <div>
```

Comment:
Same as with the other SVG, the `div` is not needed here, the `svg` tag is fine on its own :)